src/test/java/ioananadroflorea/tests/Tema7/BaseTest.java

```diff
@@ -1,6 +1,6 @@
 package ioananadroflorea.tests.Tema7;
 
-import ioanaandroflorea.driver.BrowserManager;
```

Comment:
Change ioanaandroflorea to ioanandroflorea and remove the import for your package. Thanks!